test/prow-cluster-operation/actions/delete.go

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package actions
+
+import (
+	"log"
+
+	"knative.dev/pkg/testutils/clustermanager"
+	"knative.dev/pkg/testutils/common"
+	"knative.dev/test-infra/test/prow-cluster-operation/options"
+)
+
+func Delete(o *options.RequestWrapper) {
+	o.Request.NeedsCleanup = true
+	o.Request.SkipCreation = true
+
+	gkeClient := clustermanager.GKEClient{}
+	clusterOps := gkeClient.Setup(o.Request)
+	gkeOps := clusterOps.(*clustermanager.GKECluster)
+	if err := gkeOps.Acquire(); err != nil || gkeOps.Cluster == nil {
+		log.Fatalf("Failed identifying cluster for cleanup: '%v'", err)
+	}
+	log.Printf("Identified project %q and cluster %q for removal", gkeOps.Project, gkeOps.Cluster.Name)
+	// Don't wait for delete
```

Comment:
It seems the comment is not correct, since the `Delete` function is a blocking operation.